From 6da84d8a11aa22c4189da231b04f77f8c43bd3e1 Mon Sep 17 00:00:00 2001
From: Steve Dickson <SteveD@redhat.com>
Date: Thu, 18 Sep 2014 14:19:32 -0400
Subject: [fs] nfs: get rid of duplicate dprintk

Message-id: <1411050057-12417-10-git-send-email-steved@redhat.com>
Patchwork-id: 96185
O-Subject: [RHEL7.1 PATCH 09/94] nfs: get rid of duplicate dprintk
Bugzilla: 1111708
RH-Acked-by: J. Bruce Fields <bfields@redhat.com>
RH-Acked-by: Benjamin Coddington <bcodding@redhat.com>

From: Weston Andros Adamson <dros@primarydata.com>

This was introduced by a merge error with my recent pgio patchset.

Signed-off-by: Weston Andros Adamson <dros@primarydata.com>
Signed-off-by: Trond Myklebust <trond.myklebust@primarydata.com>
Upstream Id: 0446278999e5b622b309179ba392efcb5d8b8a46
Bugzilla: 1111708
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/fs/nfs/read.c b/fs/nfs/read.c
index b1532b7..beff276 100644
--- a/fs/nfs/read.c
+++ b/fs/nfs/read.c
@@ -115,12 +115,6 @@ static void nfs_readpage_release(struct nfs_page *req)
 
   unlock_page(req->wb_page);
  }
-
- dprintk("NFS: read done (%s/%Lu %d@%Ld)\n",
-   req->wb_context->dentry->d_inode->i_sb->s_id,
-   (unsigned long long)NFS_FILEID(req->wb_context->dentry->d_inode),
-   req->wb_bytes,
-   (long long)req_offset(req));
  nfs_release_request(req);
 }
 
-- 
1.7.1